assembl/scripts/db_manage.py

```diff
@@ -33,10 +33,11 @@ def main():
     configure_indexing()
     engine = configure_engine(settings, True)
     if args.command == "bootstrap":
-        db = bootstrap_db(args.configuration)
-        bootstrap_db_data(db)
-        mark_changed()
-        transaction.commit()
+        with transaction.manager:
+            db = bootstrap_db(args.configuration)
+            mark_changed()
+        with transaction.manager:
+            bootstrap_db_data(db)
```

Comment:
Is there a reason you do two commits. If yes, can you please add the reason in a comment.